Homework_8_MyAmazingRPG/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,66 @@
+public class Character {
+    String name;                    //Ім'я персонажа.
+    int health;                     //Кількість очок здоров'я персонажа.
+    int strength;                   //Атрибут сили, який впливає на силу атаки.
+    int defense;                    //Атрибут захисту, який впливає на зменшення шкоди, коли об'єкт захищається.
+    static int characterCount;      //Статичне поле для відстеження кількості створених персонажів.
+
+    Character(String name, int health, int strength, int defense) {
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
+        this.defense = defense;
+        characterCount++;
+    }
+
+
+
+    public int Attack(Character otherPerson) {
```

Comment:
Non-static methods of the class - names should start with a lowercase letter according to convention

`public int attack`

Only constructor methods start with capital letter (as they have to be named identically to how their class is named)


https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html